lib/test_track_rails_client/rspec_helpers.rb

```diff
@@ -2,4 +2,8 @@
 
 RSpec.configure do |config|
   config.include TestTrackRailsClient::AssignmentHelper
+
+  config.before(:each) do
```

Comment:
regarding loading the schema file before each spec, seems that we could do something like make Schema a singleton that loads the file once, this line becomes a `config.before(:all)`, `stub_test_track_assignments` mutates an in memory registry, and in a `config.before(:each)` we reset it back to the default.